roles/build-release/tasks/tests.yaml

```diff
@@ -0,0 +1,94 @@
+---
+- name: Clean up virtualenv if necessary
+  ansible.builtin.file:
+    path: "{{ antsibull_ansible_venv }}"
+    state: absent
+  when: antsibull_venv_cleanup | bool
+
+- name: Get Ansible python fact
+  setup:
+    filter: ansible_python
+
+- name: Install the release tarball in a virtualenv so we can test it
+  ansible.builtin.pip:
+    name: "file://{{ _release_archive }}"
+    state: present
+    virtualenv: "{{ antsibull_ansible_venv }}"
+    virtualenv_command: "{{ ansible_python.executable }} -m venv"
+
+# Note: the version of ansible-base doesn't necessarily match the deps file since the version requirement is >=
+- block:
+    - name: Retrieve the expected minimum version of ansible-base
+      shell: >-
+        grep _ansible_base_version {{ antsibull_data_dir }}/{{ _deps_file }} | awk '{print $2}'
+      changed_when: false
+      register: _expected_ansible_base
+
+    - name: Retrieve the installed version of ansible-base
+      shell: >-
+        {{ antsibull_ansible_venv }}/bin/pip show ansible-base | awk '/Version/ {print $2}'
```

Comment:
Do you plan to take into account that `ansible-base` got renamed to `ansible-core` in `ansible/ansible@devel`?

Reply:
@webknjaz yup -- that'll be for 2.11 sometime in late april right ? I guess we can have a condition or a ternary that sets the name as ansible-core for >=2.11

Reply:
Yea.. The feature freeze is in like two weeks so there will probably be some betas way earlier than April.

Reply:
This also needs some code changes in antsibull itself. The first Ansible 4 beta is planned for March 3rd, so we have a bit more time for these changes :)

Reply:
Ansible or ansible-base/core?

Reply:
Ansible: https://hackmd.io/y7BBcweNR3aRVLuMbKkDxw#Proposed-Extremely-Tentative-Schedule-for-Ansible-400

Reply:
but we're talking about `pip show ansible-base` here

Reply:
Ansible 4.0.0 is the first Ansible version which needs ansible-core. So until we start working on that, `pip show ansible-base` is fine.